### ISSUE TITLE
false rewritten as False in elasticsearch.yml

### ISSUE THREAD

**Issue by babadofar** [Traackr/ansible-elasticsearch#40]
a little gotcha in the elasticsearch.yml configuration. In your examples you always have "true" and "false" in quotes, but should you happen to put true and false without quotes, it ends up in the elasticsearch.yml file as False with a capital first letter, and elasticsearch doesn't understand that.

**Comment by kazgurs** [comment]
True, more generally a YAML gotcha with boolean conversion. When you want literal values as a string, you must always put them in quotes:
```
non_boolean: "yes"
other_string: "false"
```
